src/julia/src/CommonWorm.jl

```diff
@@ -115,10 +114,18 @@ function parsed_json_to_worm(d :: Dict{AbstractString, Any})
     end
     for i in 1:length(x)
         if length(x[i]) != length(y[i])
-            result = string("Failed to parse worm ", id, " lengths of x and y spines do not agree at t = ", t[i])
+            result = string("Failed to parse worm ", id, ": lengths of x and y spines do not agree at t = ", t[i])
             return result
         end
     end
+    if (haskey(d, "cx") != haskey(d, "cy"))
```

Comment:
Huh, I did that because I though it was the _most_ obvious!  (Also, because `^` means exponentiation in Julia.)